src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,17 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+import { User } from '../../types/User';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const { name, email } = user;
+
+  return (
+    <>
+      <a className="UserInfo" href={`mailto:${email}`}>
+        {name}
+      </a>
+    </>
```

Comment:
it's a redundant tag wrapper
```suggestion
      <a className="UserInfo" href={`mailto:${email}`}>
        {name}
      </a>
```